jdk/src/share/bin/java.c

```diff
@@ -297,29 +335,23 @@ JLI_Launch(int argc, char ** argv,              /* main argc, argc */
         SetClassPath(cpath);
     }
 
+    /* Process -Xmso to set the main thread stack size. May be overridden by
+     * a later command line option.
+     */
     {
-        /* Process -Xmso in the OPENJ9_JAVA_OPTIONS environment variable to
-         * set the main thread stack size. May be overridden by a later command
-         * line option.
-         */
-        JLI_List openj9Args = JLI_List_new(8); /* 8 is arbitrary */
-        if (JLI_ParseOpenJ9ArgsFromEnvVar(openj9Args, "OPENJ9_JAVA_OPTIONS")) {
-            size_t i = openj9Args->size;
-            while (i > 0) {
-                i -= 1;
-                if (JLI_StrCCmp(openj9Args->elements[i], "-Xmso") == 0) {
-                    jlong tmp = 0;
-                    if (parse_size(openj9Args->elements[i] + 5, &tmp)) {
-                        threadStackSize = tmp;
-                        if (threadStackSize > 0 && threadStackSize < (jlong)STACK_SIZE_MINIMUM) {
-                            threadStackSize = STACK_SIZE_MINIMUM;
-                        }
-                    }
-                    break;
-                }
-            }
-            JLI_List_free(openj9Args);
-        }
+#if defined(_WIN32)
+#define OPTIONS_DEFAULT_PATH "/bin/" OPENJ9_LIBS_SUBDIR "/options.default"
+#elif defined(MACOSX) /* _WIN32 */
+#define OPTIONS_DEFAULT_PATH "/lib/" OPENJ9_LIBS_SUBDIR "/options.default"
+#else /* _WIN32 */
+#define OPTIONS_DEFAULT_PATH "/lib/" LIBARCHNAME "/" OPENJ9_LIBS_SUBDIR "/options.default"
+#endif /* _WIN32 */
```

Comment:
The path separator on Windows is `\`, and the comments should repeat the previous condition:
```c
#if defined(_WIN32)
#define OPTIONS_DEFAULT_PATH "\\bin\\" OPENJ9_LIBS_SUBDIR "\\options.default"
#elif defined(MACOSX) /* defined(_WIN32) */
#define OPTIONS_DEFAULT_PATH "/lib/" OPENJ9_LIBS_SUBDIR "/options.default"
#else /* defined(MACOSX) */
#define OPTIONS_DEFAULT_PATH "/lib/" LIBARCHNAME "/" OPENJ9_LIBS_SUBDIR "/options.default"
#endif /* defined(_WIN32) */
```

Reply:
Updated.